store/mockstore/mocktikv/mvcc_leveldb.go

```diff
@@ -683,6 +686,28 @@ func checkConflictValue(iter *Iterator, m *kvrpcpb.Mutation, startTS uint64) err
 			}
 		}
 	}
+	if startTS > 0 {
+		// Check if rollback write record(startTS, startTS, typeRollback) exists on key
+		info, ok, err1 := getTxnCommitInfo(iter, m.Key, startTS)
+		if err1 != nil {
+			err = errors.Trace(err1)
+			return err
+		}
+		if ok && info.commitTS == startTS {
+			logutil.BgLogger().Warn("rollback value found",
+				zap.Uint64("txnID", startTS),
+				zap.Int32("info.valueType", int32(info.valueType)),
```

Comment:
it seems this value always be `typeRollback`, maybe better change `info.*` to `rollbacked.*`~?

and add type info to above panic message `panic(fmt.Sprintf("invalid value type %v not rollback", info.valueType))`